test/stache-test.js

```diff
@@ -6271,6 +6271,12 @@ function makeTest(name, doc, mutation) {
 		equal(view.firstChild.lastChild.nodeValue, "yep", "Got aliased value");
 	});
 
+	test('check if <content> is already registred #165', function () {
```

Comment:
I think to test this you'll need to register a `content` tag and then use `steal-clone` to re-import `can-stache` and make sure there is not  a warning given that says `content is already defined`.